src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import core.basesyntax.exception.FruitException;
+import org.junit.Test;
+
+public class FruitTransactionTest {
```

Comment:
models are not tested usually, so most of these tests are not needed, we could have a `getByCode()` tested, but that's it.|
You can still leave them if you want

Reply:
I did it as a super task, I wanted 100% code coverage)